index.js

```diff
@@ -45,10 +48,10 @@ function preset(_, explicitOptions = {}) {
     node: env === 'production' ? '8.3' : 'current',
   };
 
-  const webTargets = {
-    browsers:
-      env === 'production' ? options.browsers : ['last 2 Chrome versions'],
-  };
+  const webTargets =
+    env === 'production'
+      ? { browsers: options.browsers }
+      : { esmodules: true };
```

Comment:
this is a neat addition, it only selects browsers that support modules, which is a good signpost for "not much to compile" but still covers chrome, ff, edge, and safari